spec/services/dttp/contact_update_spec.rb

```diff
@@ -30,11 +30,20 @@ module Dttp
         let(:placement_response) { double(code: 204) }
 
         it "sends a PATCH request to update contact and placement assignment entities" do
+          allow(CreateOrUpdateConsistencyCheckJob).to receive(:perform_later).and_return(true)
           expect(Client).to receive(:patch).with(contact_path, body: contact_payload).and_return(contact_response)
           expect(Client).to receive(:patch).with(placement_path, body: placement_payload).and_return(placement_response)
 
           described_class.call(trainee: trainee)
         end
+
+        it "enqueues the CreateOrUpdateConsistencyJob" do
+          allow(Client).to receive(:patch).with(contact_path, body: contact_payload).and_return(contact_response)
```

Comment:
Can we extract these into a before? They're duplicated in the previous spec. I think this is what Sonarcloud is complaining about too. (same in the other specs)